### ISSUE TITLE
Build warning (single one multiple times) regarding unused result

### ISSUE THREAD

**Issue by falkTX** [sadko4u/lsp-plugins#37]
Building lsp-plugins I get a build warning repeated several times.

```
/Shared/Personal/FOSS/Repo/Debian-Plugins/lsp-plugins-src-1.1.7/include/container/jack/main/posix.h: In function ‘int (* lsp::lookup_jack_main(void**, const char*))(const char*, int, const char**)’:
/Shared/Personal/FOSS/Repo/Debian-Plugins/lsp-plugins-src-1.1.7/include/container/jack/main/posix.h:78:73: warning: ignoring return value of ‘int asprintf(char**, const char*, ...)’, declared with attribute warn_unused_result [-Wunused-result]
             asprintf(&ptr, "%s" FILE_SEPARATOR_S "%s", path, de->d_name);
```

This spams the build log, there's really a lot of them..

Using Ubuntu 14.04, gcc-4.8.5 (for generic kxstudio packages)

**Comment by sadko4u** [comment]
This commit should fix all these warnings: 3009e53cd3a3821fab147da8a7c57609bc4e40bd

**Comment by sadko4u** [comment]
Merged change into **devel** branch: 6830de7d3fb050a91ac81c18ff0e1d125c4522c5

**Comment by sadko4u** [comment]
Release 1.1.9 has been issued.
